ui/components/component-library/avatar-network/avatar-network.scss

```diff
@@ -1,10 +1,21 @@
+[data-theme='dark'],
+.dark {
+  .mm-avatar-network {
+    border-color: var(--color-background-default);
+  }
+}
+
```

Comment:
I think using the defaults [here](https://github.com/MetaMask/metamask-extension/pull/28976/files#diff-13e7c70fab80ac835b995a5ec7b836b03fa9681244c939e764ea85e75b5a741dR26) would do the same thing?
```
borderColor = BorderColor.backgroundDefault,
```
```suggestion
```